pages/mesosphere/dcos/services/elastic/2.7.0-6.8.1/how-to-guides/index.md

```diff
@@ -12,7 +12,7 @@ render: mustache
 
 ## 1. Specify the name and settings for your Elastic and Kibana services
 
-These will be used in subsequent commands. If you have existing Elastic or Kibana configuration JSON file, the values should match.
+The names will be used in subsequent commands. If you have an existing Elastic or Kibana configuration JSON file, the values should match.
```

Comment:
@cmastr it's not just "names", it's "name and settings" (as specified in the heading).